home/templates/home/blocks/highlighted_column_block.html

```diff
@@ -0,0 +1,85 @@
+{% load i18n wagtailcore_tags %}
+
+<!-- single column template checks to ensure doesn't dispay additional columns -->
+{% if self.column_count == "12" %}
+  {% for item in self.column %}
+    {% if forloop.counter <= 2 %}
```

Comment:
you need <2 or <=1 here

Reply:
Good point! although, after testing I will actully use splice in the forloop instead of this method

Reply:
Ok, so I merge it like this and you change it in the other branch or whats the plan?